### ISSUE TITLE
Docs aren't published since v4.4.0

### ISSUE THREAD

**Issue by jelhan** [kaliber5/ember-bootstrap#1387]
The docs hosted at https://www.ember-bootstrap.com/ haven't been published anymore since v4.4.0:

```js
// developers tools when running https://www.ember-bootstrap.com/
JSON.parse(
  decodeURIComponent(
    document.querySelector('meta[name$="config/environment"]').content
  )
).APP.version // "v4.4.0+19ded703"
```

The build is timing out in CI:

- https://github.com/kaliber5/ember-bootstrap/runs/1493307110 (v4.5.0)
- https://github.com/kaliber5/ember-bootstrap/runs/1627376802 (v4.6.0)
- https://github.com/kaliber5/ember-bootstrap/runs/1664220607 (v4.6.0 restarted just know)

I tried to run `yarn run docs:build` locally but also got stucked with the same output.

**Comment by simonihmig** [comment]
Thanks for reporting, will try to look into this...

**Comment by jelhan** [comment]
Should be fixed by #1514. But I would propose to keep this one open until it has been deployed successfully.

**Comment by simonihmig** [comment]
Just released `v4.7.0`, and docs were successfully deployed. So closing this.

Unfortunately there is a regression, also visible in the docs, will try to fix this asap: #1518.
